oaknational/Oak-Web-Application
# Feat/pupil 684/move pages away from beta

## Description

Music year: 1983

Moved beta pages to directly under pupil.
Updated urls for the navigation buttons within the browse pages.
Updated unit tests with new urls and imports. 

Fixes #

## How to test

1. Go to https://deploy-preview-2450--oak-web-application.netlify.thenational.academy/pupils/years
2. You should see the year selection for pupils
3. Check that you can browse through the years, subjects, options & lessons without any issues.

## Checklist
